src/com/rcv/Logger.java

```diff
@@ -20,95 +29,96 @@
-      Logger.log("Couldn't remove logger:%s",loggerOutputPath);
-    }
+  // function: warn
+  // purpose: log formatted output to console and audit file at WARNING level
+  // param: format string into which object params will be formatted
+  // param: obj object to be parsed into format string
+  static void warn(String format, Object... obj) {
+    tabulationLogger.warning(String.format(format, obj));
   }
 
+  // function: severe
+  // purpose: log formatted output to console and audit file at SEVERE level
+  // param: format string into which object params will be formatted
+  // param: obj object to be parsed into format string
+  static void severe(String format, Object... obj) {
+    tabulationLogger.severe(String.format(format, obj));
+  }
 
-  // function: addLogger
-  // purpose: create and add a new logger object
-  // param: loggerOutputPath: file path for logging output
-  // param: logToConsole: weather to log to console or not
-  // file access: write (existing file will be overwritten)
+  // function: addTabulationFileLogging
+  // purpose: adds file logging for a tabulation run
+  // param: loggerOutputPath: file path for tabulationLogger logging output
+  // file access: write - existing file will be overwritten
   // throws: IOException if unable to open loggerOutputPath
-  static void addLogger(String loggerOutputPath, Boolean logToConsole) throws IOException {
-    java.util.logging.Logger logger = loggers.get(loggerOutputPath);
-    if(logger != null) {
-      Logger.log("Logger has already been added:%s",loggerOutputPath);
-    } else {
-
-      // specifies how logging output lines should appear
-      LogFormatter formatter = new LogFormatter();
-
-      // get or create the logger
-      logger = java.util.logging.Logger.getLogger(loggerOutputPath);
-      // remove any existing handlers to prevent duplicate logging
-      for (Handler h : logger.getHandlers()) {
-        logger.removeHandler(h);
-      }
-
-      // add handlers
-      if(logToConsole) {
-        // create new handler for console logging and add it
-        ConsoleHandler consoleHandler = new ConsoleHandler();
-        consoleHandler.setFormatter(formatter);
-        logger.addHandler(consoleHandler);
-      }
-
-      // create new handler for file logging and add it
-      FileHandler fileHandler = new FileHandler(loggerOutputPath);
-      fileHandler.setFormatter(formatter);
-      logger.addHandler(fileHandler);
-
-      // dont log to default logger
-      logger.setUseParentHandlers(false);
+  static void addTabulationFileLogging(String loggerOutputPath) throws IOException {
+    // create new handler for file logging and add it
+    FileHandler fileHandler = new FileHandler(loggerOutputPath);
+    // specifies how logging output lines should appear
+    LogFormatter formatter = new LogFormatter();
+    fileHandler.setFormatter(formatter);
+    // get the tabulationLogger logger and add file handler
+    tabulationLogger.addHandler(fileHandler);
```

Comment:
Log a message here that a new tabulation logger was successfully created?

Reply:
yep.  added messages for tabulation logging start / stop